available-connectors.json

```diff
@@ -48,12 +48,8 @@
       "port": {
         "type": "number"
       },
-      "user": {
-        "type": "string"
-      },
       "password": {
-        "type": "string",
-        "display": "password"
+        "type": "string"
```

Comment:
Oh, I think I was wrong here. I see that other connectors use `"display": "password"` too. Is this perhaps referring to Inquirer prompt types? 

https://www.npmjs.com/package/inquirer#password---type-password

<img width="517" alt="screen shot 2016-10-18 at 14 52 03" src="https://cloud.githubusercontent.com/assets/1140553/19478330/74dca5a8-9542-11e6-8c84-b3b0bbc91c2b.png">

@superkhau could you please check?